data-explorer/kusto/query/printoperator.md

```diff
@@ -25,7 +25,7 @@ print x=1, s=strcat("Hello", ", ", "World!")
 
 ## Returns
 
-A single-column, single-row, table whose single cell has the value of the evaluated *ScalarExpression*.
+A one or more column, single-row, table whose single cell has the value of the evaluated *ScalarExpression*.
```

Comment:
Needs rephrasing, it's not a "single cell" anymore :)
By the way, the beginning of the sentence is "A one or more column" sounds a bit strange (though grammatically it's probably correct).